source/index.twig

```diff
@@ -1,54 +1,59 @@
-
-                            <div class="clearfix"></div>
-
-                            <h1 class="header_title">
-                                {{ copy.killer_headline }}
-                            </h1>
-
-                            <a href="#show" class="btn btn-custom mt-5">
-                                <em class="pe-7s-play pe-lg"></em>
-                                &nbsp;
-                                Oh, Really?
-                            </a>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </section>
-
-        {% include "_snippets/section/projects.twig" %}
-
-        {% include "_snippets/section/left_and_right.twig" %}
-
-        {% include "_snippets/section/video.twig" %}
-
-        {% include "_snippets/section/references.twig" %}
-
-        <br><br>
-
-        {% include "_snippets/layout/footer.twig" %}
-
-        {% include "_snippets/layout/google_analytics.twig" %}
-    </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        {% include "_snippets/layout/head.twig" %}
+    </head>
+
+    <body>
+        <section class="home-bg-img h-100vh">
+            <div class="container">
+                <div class="row justify-content-end">
+                    <div class="col-12 col-sm-12 col-md-6 col-lg-6">
+                        <br>
+                        <a href="/" id="main_logo">
+                            <img src="/assets/images/logo/rector-no_frame_vector.svg" alt="">
+                            <h1>Rector</h1>
+                        </a>
+
+                        <div class="clearfix"></div>
+
+                        <div class="text-white">
+                            <div class="home_divider mb-4"></div>
+
+                            <div class="clearfix"></div>
+
+                            <h1 class="header_title">
+                                {{ copy.killer_headline }}
+                            </h1>
+
+                            <a href="#show" class="btn btn-custom mt-5">
+                                <em class="pe-7s-play pe-lg"></em>
+                                &nbsp;
+                                Oh, Really?
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        {% include "_snippets/section/projects.twig" %}
+
+        {% include "_snippets/section/left_and_right.twig" %}
+
+        {% include "_snippets/section/video.twig" %}
+
+        {% include "_snippets/section/references.twig" %}
+
+        <p class="text-center">
+            <a href="{{ contact_link }}" class="btn btn-custom mt-5">
+                <em class="pe-7s-mail pe-lg"></em>&nbsp;
+                I want to refactor my code too!
+            </a>
+        </p>
+
+        {% include "_snippets/layout/footer.twig" %}
+
+        {% include "_snippets/layout/google_analytics.twig" %}
+    </body>
+</html>
```

Comment:
What changed here? There is no diff neither :/

Reply:
I added
```twig
<p class="text-center">
            <a href="{{ contact_link }}" class="btn btn-custom mt-5">
                <em class="pe-7s-mail pe-lg"></em>&nbsp;
                I want to refactor my code too!
            </a>
        </p>
```

I am sorry, i do not know why there is diff like this, maybe i messed up lineendings or spaces/tabs or any other idea?

Reply:
Spaces or line-ending was my idea too. Has something changed on your computer?

If you know what you changed, please commit it again and `force-push`. We need correct history.